src/MockClassifier.Api/Services/Dmr/DmrService.cs

```diff
@@ -40,17 +50,31 @@ public async Task ProcessRequestsAsync()
             {
                 try
                 {
-                    var response = await httpClient.PostAsJsonAsync("/", request).ConfigureAwait(true);
-                    _ = response.EnsureSuccessStatusCode();
+                    // Setup content
+                    var jsonPayload = JsonSerializer.Serialize(request.Payload);
+                    var jsonPayloadBase64 = _encodingService.EncodeBase64(jsonPayload);
+                    using var content = new StringContent(jsonPayloadBase64, Encoding.UTF8, MediaTypeNames.Application.Json);
 
+                    // Setup message
+                    using var requestMessage = new HttpRequestMessage()
```

Comment:
Might be worth moving this requestMessage creation logic into a separate method

Reply:
Yup, good spot. I've moved it into another method.